apps/pages/templates/pages/review_guide.html

```diff
@@ -83,7 +80,7 @@ <h2 id="tips">{{ _('Some tips for writing a great review') }}</h2>
     <ul>
       <li>
         {% trans %}
-          Submit one-word reviews such as "Great!", "wonderful," or "bad."
+          Submit one-word reviews such as "Great!", "Wonderful", or "Bad".
```

Comment:
I'm not sure about that... I think it's some kind of grammatical oddity, let's double check with @atsay because i'm pretty sure she had that kind of feedback the first time she proposed the modifications.